lib/manageiq/floe/workflow/runner/kubernetes.rb

```diff
@@ -3,6 +3,36 @@ module Floe
     class Workflow
       class Runner
         class Kubernetes < ManageIQ::Floe::Workflow::Runner
+          def initialize(*)
+            require "awesome_spawn"
+            require "securerandom"
+
+            super
+          end
+
+          def run!(resource, env = {}, secrets = {})
+            raise ArgumentError, "Invalid resource" unless resource&.start_with?("docker://")
+
+            image = resource.gsub("docker://", "")
```

Comment:
Super-minor, but prefer `sub` over `gsub`, since it will stop after the first replacement

Reply:
Both of these comments are pertinent to the existing runners so I opened a PR to fix the docker/podman runners first, https://github.com/agrare/manageiq-floe/pull/24